code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm

```diff
@@ -10,7 +10,7 @@
 	if(!check_and_use_plasma_owner())
 		return
 
-	playsound(xeno.loc, screech_sound_effect, 75, 0, status = 0)
+	playsound(xeno.loc, predalien_roar, 75, 0, status = 0)
```

Comment:
shouldn't this be a pick incase someone actually adds more to your list

Reply:
```suggestion
	playsound(xeno.loc, pick(predalien_roar), 75, 0, status = 0)
```